x-pack/plugins/infra/server/lib/alerting/metric_threshold/metric_threshold_executor.ts

```diff
@@ -236,16 +263,18 @@ const mapToConditionsLookup = (
 
 export const createMetricThresholdExecutor = (alertUUID: string) =>
   async function({ services, params }: AlertExecutorOptions) {
-    const { criteria, groupBy, filterQuery } = params as {
+    const { criteria, groupBy, filterQuery, sourceId } = params as {
       criteria: MetricExpressionParams[];
       groupBy: string | undefined;
       filterQuery: string | undefined;
+      sourceId?: string;
```

Comment:
Out of curiosity, why do we even take a sourceId from the frontend? Is the goal here just future-proofing?

Reply:
Yeah especially in light of https://github.com/elastic/metrics/issues/15 I figured we might want to have this be configurable at some point.